scripts/post-update.sh

```diff
@@ -282,6 +282,13 @@ if (( $PRE_0_0_16 > 0 )); then
     sudo sh -c "echo '[Service]\nTimeoutStartSec=10' > /etc/systemd/system/networking.service.d/timeout.conf"
 fi
 
+# Check pre-0.0.17 to copy udev files
+PRE_0_0_17=$(( git rev-list --count --left-right 0.0.17...revert-point || echo 0 ) | cut -f1)
+
+if (( $PRE_0_0_17 > 0 )); then
+    sudo cp $HOME/companion/params/100.autopilot.rules /etc/udev/rules.d/
```

Comment:
@Williangalvani thanks!

FWIW The idea behind using `cp` instead of `>>` is also so that we can run the command at any arbitrary point without the need to check the version or current state of the file.

Reply:
I wish I had this thought earlier in this file's history..